spec/stat_calculator_spec.rb

```diff
@@ -1,6 +1,5 @@
 # spec/stat_calculator_spec.rb
-require 'rspec'
-require_relative '../lib/stat_calculator'
+require'spec_helper'
```

Comment:
make sure to add a space between require and 'spec_helper'
also require should be require_relative for this